x-pack/plugins/maps/public/routing/routes/maps_app/load_map_and_render.tsx

```diff
@@ -5,13 +5,23 @@
  */
 
 import React from 'react';
-import { MapsAppView } from '.';
-import { getMapsSavedObjectLoader } from '../../bootstrap/services/gis_map_saved_object_loader';
-import { getCoreChrome, getToasts } from '../../../kibana_services';
 import { i18n } from '@kbn/i18n';
 import { Redirect } from 'react-router-dom';
+import { AppMountParameters } from 'kibana/public';
+import { EmbeddableStateTransfer } from 'src/plugins/embeddable/public';
+import { getCoreChrome, getToasts } from '../../../kibana_services';
+import { getMapsSavedObjectLoader } from '../../bootstrap/services/gis_map_saved_object_loader';
+import { MapsAppView } from '.';
+
+interface LoadMapAndRenderProps {
```

Comment:
Mind just naming `Props`? We have been using this convection when naming interfaces fro react properties to keep things simple and consistent across components.

Reply:
Fixed this.